src/posthog-core.ts

```diff
@@ -1280,11 +1287,41 @@ export class PostHog {
      */
     resetGroups(): void {
         this.register({ $groups: {} })
+        this.resetGroupPropertiesForFlags()
 
         // If groups changed, reload feature flags.
         this.reloadFeatureFlags()
     }
 
+    /**
+     * Set override person properties for feature flags.
+     * This is used when dealing with new persons / where you don't want to wait for ingestion
+     * to update user properties.
+     */
+    personPropertiesForFlags(properties: Properties, reloadFeatureFlags = true): void {
```

Comment:
Naming on these functions could be better. They sound like getters

Reply:
`setPersonPropertiesForFlags` ?

Reply:
yep